packages/harmony/src/components/button/BaseButton/BaseButton.tsx

```diff
@@ -73,16 +108,13 @@ export const BaseButton = forwardRef<HTMLButtonElement, BaseButtonProps>(
         {...other}
       >
         {isLoading ? (
-          <LoadingSpinner
-            data-chromatic='ignore'
-            className={cn(baseStyles.spinner, styles.spinner)}
-          />
+          <LoadingSpinner data-chromatic='ignore' css={styles.icon} />
         ) : LeftIconComponent ? (
-          <LeftIconComponent className={cn(baseStyles.icon, styles.icon)} />
+          <LeftIconComponent css={{ ...iconCss, ...styles.icon }} />
```

Comment:
Let's pass arrays in? That allows for better merging if there are nested selectors for instance